<launch>
  <arg name="robot_name" default="evolo0"/>
  <arg name="broker_addr" default="localhost"/>
  <arg name="broker_port" default="1889"/>
  <!-- <arg name="broker_addr" default="20.240.40.232"/> -->
  <!-- <arg name="broker_port" default="1884"/> -->
  <arg name="context" default="waraps"/>
  <arg name="domain" default="surface"/>
  <arg name="realsim" default="real"/>

  <arg name="mqtt_params_file" default="$(find-pkg-share str_json_mqtt_bridge)/config/waraps.yaml"/>

  <node name="mqtt_bridge_cloud" pkg="str_json_mqtt_bridge" exec="waraps_bridge" output="screen" namespace="$(var robot_name)/waraps">
    <param from="$(var mqtt_params_file)"/>
    <param name="mqtt.connection.port" value="$(var broker_port)"/>
    <param name="mqtt.connection.host" value="$(var broker_addr)"/>
    <param name="mqtt.namespace" value="$(var context)/unit/$(var domain)/$(var realsim)/$(env USER nouser)_$(var robot_name)"/>
  </node>


</launch>
